mock_service/lib/datapoint.py

```diff
@@ -33,11 +33,12 @@ def __init__(
         self.is_mocked = is_mocked
         self.value_listener = value_listener
 
+    # Do we need this -> never used as of now
```

Comment:
This check **was** intended to be used whenever you find an AnimationAction on a discrete value type and throw an error. Probably this slipped through when I released the initial version.